App/PHP_HTML/question_submission.php

```diff
@@ -15,18 +15,19 @@
 
 if (isset($_SESSION['auth'])) {
     $title = $_POST["title"];
-    $details = $_POST["details"];
+    $details = mysqli_real_escape_string($db, $_POST["details"]);
     $user_id = $_SESSION['user_id'];
     $tags = $_POST["tags"];
     $question_by_user = $_SESSION['user_name'];
+	
     $sql = "INSERT INTO questions (question_by, question_title, question_description, question_date, question_by_user) VALUES ($user_id, \"$title\", \"$details\", NOW(), '$question_by_user')";
     $db->query($sql);
 
     // increase users question count
     $sql = "UPDATE users SET user_questions_count = user_questions_count + 1 WHERE user_id = '".$user_id."'";
     $db->query($sql);
 
-    $sql = "SELECT question_id FROM questions WHERE question_by='$user_id' AND question_title = '$title'";
+    $sql = "SELECT max(question_id) FROM questions WHERE question_by='$user_id' AND question_title = '$title'";
```

Comment:
Could you explain this?

Reply:
If we create a question with title matching an existing question, then the old question will be displayed. The max function will show the latest one we created.